core/server/src/test/java/alluxio/master/file/PermissionCheckerTest.java

```diff
@@ -183,11 +185,13 @@ public static void beforeClass() throws Exception {
 
   @AfterClass
   public static void afterClass() throws Exception {
```

Comment:
I'd also prefer `@Before` vs `@BeforeClass` so that tests are less likely to interfere with each other.